lib/sycamore/sycamore/transforms/text_extraction/pdf_miner.py

```diff
@@ -88,6 +88,20 @@ def extract_document(self, filename: str, hash_key: str, use_cache=False, **kwar
                 pdf_miner_cache.set(hash_key, pages)
             return pages
 
+    @staticmethod
+    def _parse_obj(objs):
```

Comment:
Can you rename this to `get_font_size` or something like it? `parse_obj` seems inaccurate for the functionality here.